app/jobs/regular/chat_notify_mentioned.rb

```diff
@@ -2,14 +2,14 @@
 
 module Jobs
   class ChatNotifyMentioned < ::Jobs::Base
+    delegate :direct_message_channel?, to: :@chat_channel
```

Comment:
Im not a super fan of this :p Is there a reason you did it explicitly for this one?

Reply:
I thought it was weird to store `@is_direct_message_channel` in a variable, don't feel super strong tho I can just revert

Reply:
It's not possible to just do `@chat_channel.direct_message_channel?` ?

Reply:
Yah, will change it to that.